packages/mdl-snackbar/mdl-snackbar.scss

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "mdl-animation/mixins";
+@import "mdl-theme/mixins";
+@import "mdl-typography/mixins";
+@import "./variables";
+
+/* stylelint-disable scss/dollar-variable-pattern, scss/at-mixin-pattern */
+@mixin _mdl-snackbar-reflexive-margin($default-side, $value, $reset-opposite-margin: true) {
+  @if (index((right, left), $default-side) == null) {
+    @error "Invalid default margin side #{default-side}. Please specifiy either right or left";
+  }
+
+  $rtl-side: left;
+
+  @if ($default-side == left) {
+    $rtl-side: right;
+  }
+
+  margin-#{$default-side}: $value;
+  [dir="rtl"] & {
+    margin-#{$rtl-side}: $value;
+
+    @if ($reset-opposite-margin) {
+      margin-#{$default-side}: 0;
+    }
+  }
+}
```

Comment:
could you add a `/* stylelint-enable scss/dollar-variable-pattern, scss/at-mixin-pattern */` comment directive here? This way it won't disable the checks for the rest of the file.